locales/en/kafkaoverview-v2.json

```diff
@@ -5,15 +5,13 @@
   "contactSalesCardMainText": "Contact sales to get a pre-paid subscription or modify limits of your current subscription.",
   "contactSalesCardTitle": "Contact sales",
   "dataTransfer": "Data transfer",
+  "dataTransfer_price": "$0.09 / GB",
   "firstStreamingUnit": "1 streaming unit",
-  "heroCallToActionButton": "",
   "heroDescription": "Red Hat OpenShift Streams for Apache Kafka is a managed cloud service that provides a streamlined developer experience for building, deploying, and scaling real-time applications in hybrid-cloud environments. The combination of seamless operations across distributed microservices, large data transfer volumes, and managed operations allows teams to focus on core competencies, accelerate application velocity and reduce operational cost.",
   "heroTagline": "Fully hosted and managed service for stream-based applications",
   "heroTitle": "Get started with Red Hat OpenShift Streams for Apache Kafka",
-  "heroTryItButton": "",
   "kafkaInstanceCapacityDescription": "OpenShift Streams offers instances in the following sizes with the following capacity.",
-  "kafkaInstanceCapacityfooter1": "*Storage can be extended. If you wish to increase the storage limit for a Kafka instance, contact",
-  "kafkaInstanceCapacityfooter2": "to discuss your options. You can check your current storage usage by",
+  "kafkaInstanceCapacityfooter": "*Storage can be extended. If you wish to increase the storage limit for a Kafka instance, contact <0>Red Hat support</0> to discuss your options. You can check your current storage usage by <1>monitoring disk space metrics</1>.",
```

Comment:
@HemaHG here the copy is complete with all the elements. We then inject the buttons with the right styling and attributes in the code.